src/frontend/components/Felleskomponenter/Datovelger/Datovelger.tsx

```diff
@@ -122,7 +102,7 @@ const Datovelger: React.FC<DatoVelgerProps> = ({
                 feil={!!(felt.feilmelding && skjema.visFeilmeldinger)}
                 disabled={disabled}
                 locale={valgtLocale}
-                allowNavigationToDisabledMonths={true}
+                allowNavigationToDisabledMonths={false}
```

Comment:
Hvorfor endre denne? Det gjør at hvis første dato gyldige dato er den 1. i inneværende måte kan man trykke på "forrige måned"-knappen, men ingenting skjer siden det ikke er noen gyldige datoer der, uten noen form for tilbakemelding til søker. Samme for fremover i tid hvis siste gyldige dato er den siste dagen i inneværende måned, men det tror jeg ikke vi har noen caser på.